cmd/pdbinspect/main.go

```diff
@@ -0,0 +1,170 @@
+// Copyright (C) 2018 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package main
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"os"
+	"time"
+
+	"github.com/gogo/protobuf/jsonpb"
+	"github.com/gogo/protobuf/proto"
+	"github.com/golang/protobuf/ptypes/timestamp"
+	"github.com/spf13/cobra"
+
+	"storj.io/storj/pkg/pb"
+	"storj.io/storj/pkg/pointerdb/pdbclient"
+	"storj.io/storj/pkg/provider"
+	"storj.io/storj/pkg/storage/meta"
+)
+
+func main() {
+	ctx := context.Background()
+	var port string
+	var apiKey string
+	var prefix string
+	var endBefore string
+	var startAfter string
+	var recursive bool
+	var limit int
+	var metaFlags uint32
+	var jsonBool bool
+
+	var cmdList = &cobra.Command{
+		Use:   "list",
+		Short: "lists pointers",
+		Run: func(cmd *cobra.Command, args []string) {
```

Comment:
nit - for readability it might make sense to move this func (and the one for Get) to the bottom so that up here you can just have:

```
var cmdList = &cobra.Command{
    Use: "list",
    Short: "lists pointers",
    Run: List,
}
...
...
func List(cmd *cobra.Command, args[]string) {
...
}
```

Reply:
^ the above would also make it easier to integrate it into the inspector cli tool in the future